[Bug Skip]: [Regression]UT failures 2026-6-28

### 🐛 Describe the bug with skip template

Cases:
~~op_ut,third_party.torch-xpu-ops.test.xpu.export.test_retraceability_xpu.RetraceExportTestExport,test_dynamic_lstm_retraceability_strict~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.functorch.test_control_flow_xpu.TestControlFlowTraced,test_while_loop_autograd_simple~~
~~op_ut,third_party.torch-xpu-ops.test.xpu.higher_order_ops.test_invoke_subgraph_xpu.TestInvokeSubgraphCompile,test_bwd_partitioning~~

## Errorlog
### torch.fx.experimental.symbolic_shapes.ConstraintViolationError: Constraints violated (L['x'].size()[0])! For more information, run with TORCH_LOGS="+dynamic".
```bash
_______ RetraceExportTestExport.test_dynamic_lstm_retraceability_strict ________
[gw4] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/export/_trace.py", line 984, in _export_to_torch_ir
    gm_torch_level = dynamo_graph_capture(*args, **kwargs)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/functional_export.py", line 1031, in inner
    _suggest_or_raise_constraint_violation(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/functional_export.py", line 511, in _suggest_or_raise_constraint_violation
    raise constraint_violation_error
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/functional_export.py", line 459, in _suggest_or_raise_constraint_violation
    graph_capture_output.graph_capture_output.build_guards(fn.__code__)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/convert_frame.py", line 1204, in build_guards
    return CheckFunctionManager(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/guards.py", line 4505, in __init__
    builder, guard_manager = self.build_guards(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/guards.py", line 4824, in build_guards
    guard.create(builder)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_guards.py", line 373, in create
    return self.create_fn(builder, self)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/guards.py", line 3330, in SHAPE_ENV
    python_code_parts, verbose_code_parts = _get_code_parts(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/_dynamo/guards.py", line 3303, in _get_code_parts
    return output_graph.shape_env.produce_guards_verbose(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/fx/experimental/symbolic_shapes.py", line 6889, in produce_guards_verbose
    raise ConstraintViolationError(
torch.fx.experimental.symbolic_shapes.ConstraintViolationError: Constraints violated (L['x'].size()[0])! For more information, run with TORCH_LOGS="+dynamic".
  - You marked L['x'].size()[0] as dynamic but your code specialized it to be a constant (32). If you're using mark_dynamic, either remove it or use maybe_mark_dynamic. If you're using Dim.DYNAMIC, replace it with either Dim.STATIC or Dim.AUTO.
```
### AssertionError: 'clas[647 chars]-1)\n        unsqueeze: "f32[1, 3, 3]" = torch[1255 chars],)\n' != 'clas[647 chars]-1)\n\n        unsqueeze: "f32[1, 3, 3]" = tor[1257 chars],)\n'
```bash
____________ TestControlFlowTraced.test_while_loop_autograd_simple _____________
[gw6] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/functorch/test_control_flow_xpu.py", line 8995, in test_while_loop_autograd_simple
    self.assertExpectedInline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 3533, in assertExpectedInline
    return super().assertExpectedInline(actual if isinstance(actual, str) else str(actual), expect, skip + 1)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 413, in assertExpectedInline
    assert_expected_inline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 378, in assert_expected_inline
    assert_eq(expect, actual, msg=help_text)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 450, in assertMultiLineEqualMaybeCppStack
    self.assertMultiLineEqual(expect, actual, *args, **kwargs)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 1226, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: 'clas[647 chars]-1)\n        unsqueeze: "f32[1, 3, 3]" = torch[1255 chars],)\n' != 'clas[647 chars]-1)\n\n        unsqueeze: "f32[1, 3, 3]" = tor[1257 chars],)\n'
  class GraphModule(torch.nn.Module):
      def forward(self, primals_1: "f32[3, 3]", primals_2: "f32[3, 3]", primals_3: "f32[3]"):
          while_loop_cond_graph_0 = self.while_loop_cond_graph_0
          while_loop_body_graph_0 = self.while_loop_body_graph_0
          while_loop_stack_output = torch.ops.higher_order.while_loop_stack_output(while_loop_cond_graph_0, while_loop_body_graph_0, (primals_1,), (primals_3, primals_2));  while_loop_cond_graph_0 = while_loop_body_graph_0 = None
          getitem: "f32[u2, 3, 3]" = while_loop_stack_output[0];  while_loop_stack_output = None
          select: "f32[3, 3]" = torch.ops.aten.select.int(getitem, 0, -1)
+ 
          unsqueeze: "f32[1, 3, 3]" = torch.ops.aten.unsqueeze.default(primals_1, 0);  primals_1 = None
          slice_1: "f32[u2 - 1, 3, 3]" = torch.ops.aten.slice.Tensor(getitem, 0, 0, -1);  getitem = None
          cat: "f32[u2, 3, 3]" = torch.ops.aten.cat.default([unsqueeze, slice_1]);  unsqueeze = slice_1 = None
          return (select, primals_2, primals_3, cat)
  
      class while_loop_cond_graph_0(torch.nn.Module):
          def forward(self, arg0_1: "f32[3, 3]", arg1_1: "f32[3]", arg2_1: "f32[3, 3]"):
              sum_1: "f32[]" = torch.ops.aten.sum.default(arg0_1);  arg0_1 = None
              lt: "b8[]" = torch.ops.aten.lt.Scalar(sum_1, 2);  sum_1 = None
              return lt
  
      class while_loop_body_graph_0(torch.nn.Module):
          def forward(self, arg0_1: "f32[3, 3]", arg1_1: "f32[3]", arg2_1: "f32[3, 3]"):
              mul: "f32[3, 3]" = torch.ops.aten.mul.Tensor(arg0_1, arg0_1)
              add: "f32[3, 3]" = torch.ops.aten.add.Tensor(mul, 1);  mul = None
              t: "f32[3, 3]" = torch.ops.aten.t.default(arg2_1);  arg2_1 = None
              addmm: "f32[3, 3]" = torch.ops.aten.addmm.default(arg1_1, arg0_1, t);  arg1_1 = arg0_1 = t = None
              add_1: "f32[3, 3]" = torch.ops.aten.add.Tensor(add, addmm);  add = addmm = None
              return (add_1,)
 : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)
```
### AssertionError: 'clas[698 chars]em)\n        cos: "f32[8, 8]" = torch.ops.aten[587 chars]1)\n' != 'clas[698 chars]em)\n\n        cos: "f32[8, 8]" = torch.ops.at[589 chars]1)\n'
```bash
_______________ TestInvokeSubgraphCompile.test_bwd_partitioning ________________
[gw4] linux -- Python 3.10.20 /tmp/xpu-tool/Python/3.10.20/x64/bin/python
Traceback (most recent call last):
  File "/__w/torch-xpu-ops/torch-xpu-ops/pytorch/third_party/torch-xpu-ops/test/xpu/higher_order_ops/test_invoke_subgraph_xpu.py", line 2019, in test_bwd_partitioning
    self.assertExpectedInline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/torch/testing/_internal/common_utils.py", line 3533, in assertExpectedInline
    return super().assertExpectedInline(actual if isinstance(actual, str) else str(actual), expect, skip + 1)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 413, in assertExpectedInline
    assert_expected_inline(
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 378, in assert_expected_inline
    assert_eq(expect, actual, msg=help_text)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/site-packages/expecttest/__init__.py", line 450, in assertMultiLineEqualMaybeCppStack
    self.assertMultiLineEqual(expect, actual, *args, **kwargs)
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 1226, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
  File "/tmp/xpu-tool/Python/3.10.20/x64/lib/python3.10/unittest/case.py", line 675, in fail
    raise self.failureException(msg)
AssertionError: 'clas[698 chars]em)\n        cos: "f32[8, 8]" = torch.ops.aten[587 chars]1)\n' != 'clas[698 chars]em)\n\n        cos: "f32[8, 8]" = torch.ops.at[589 chars]1)\n'
  class GraphModule(torch.nn.Module):
      def forward(self, primals_1: "f32[8, 8]", primals_2: "f32[8, 8]"):
          partitioned_fw_subgraph_0_0 = self.partitioned_fw_subgraph_0_0
  
          invoke_subgraph_2 = torch.ops.higher_order.invoke_subgraph(partitioned_fw_subgraph_0_0, 'partitioned_fw_subgraph_0_0', primals_1, primals_2);  partitioned_fw_subgraph_0_0 = primals_1 = primals_2 = None
          getitem_6: "f32[8, 8]" = invoke_subgraph_2[3]
          getitem_5: "f32[8, 8]" = invoke_subgraph_2[2]
          getitem_4: "f32[8, 8]" = invoke_subgraph_2[1]
  
          getitem: "f32[8, 8]" = invoke_subgraph_2[0];  invoke_subgraph_2 = None
          sin: "f32[8, 8]" = torch.ops.aten.sin.default(getitem)
+ 
          cos: "f32[8, 8]" = torch.ops.aten.cos.default(getitem);  getitem = None
          return (sin, getitem_6, getitem_5, getitem_4, cos)
  
      class partitioned_fw_subgraph_0_0(torch.nn.Module):
          def forward(self, primals_0: "f32[8, 8]", primals_1: "f32[8, 8]"):
              mm: "f32[8, 8]" = torch.ops.aten.mm.default(primals_0, primals_1)
  
              sin: "f32[8, 8]" = torch.ops.aten.sin.default(mm)
  
              t: "f32[8, 8]" = torch.ops.aten.t.default(primals_0);  primals_0 = None
              t_1: "f32[8, 8]" = torch.ops.aten.t.default(primals_1);  primals_1 = None
              return (sin, mm, t, t_1)
 : To accept the new output, re-run test with envvar EXPECTTEST_ACCEPT=1 (we recommend staging/committing your changes before doing this)
```
## Pytorch Version
latest good : e683636150f35c3972bde18ac610291f3e9e577f
current : 9d7d44ef85c40965adf8f137c55ead1105f735e1

### Versions

<details>
<summary>Detail</summary>
Collecting environment information...
PyTorch version: 2.14.0a0+git9d7d44e
Is debug build: False
CUDA used to build PyTorch: None
ROCM used to build PyTorch: N/A

OS: Ubuntu 24.04.4 LTS (x86_64)
GCC version: (Ubuntu 13.3.0-6ubuntu2~24.04.1) 13.3.0
Clang version: Could not collect
CMake version: version 3.31.6
Libc version: glibc-2.39

Python version: 3.10.20 (main, Jun 16 2026, 21:01:32) [GCC 13.3.0] (64-bit runtime)
Python platform: Linux-5.15.0-176-generic-x86_64-with-glibc2.39
Is CUDA available: False
CUDA runtime version: No CUDA
CUDA_MODULE_LOADING set to: N/A
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA
Is XPU available: True
XPU used to build PyTorch: 20260000
Intel GPU driver version:
* intel-opencl-icd:	25.18.33578.51-1146~24.04
* libze1:	1.24.0.0-1146~24.04
Intel GPU models onboard:
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
* Intel(R) Data Center GPU Max 1100
Intel GPU models detected:
* [0] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-0f00-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [1] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-1600-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [2] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-1e00-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [3] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-af00-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [4] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-b300-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [5] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-bf00-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
* [6] _XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1100', platform_name='Intel(R) oneAPI Unified Runtime over Level-Zero', type='gpu', device_id=0xBDA, uuid=8680da0b-2f00-0000-c300-000000000000, driver_version='1.6.33578+51', total_memory=49136MB, local_mem_size=128KB, last_level_cache_size=196608KB, max_compute_units=448, memory_clock_rate=3200MHz, memory_bus_width=64-bit, gpu_eu_count=448, gpu_subslice_count=56, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1, is_integrated_gpu=0)
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: False
Caching allocator config: N/A

CPU:
Architecture:                            x86_64
CPU op-mode(s):                          32-bit, 64-bit
Address sizes:                           46 bits physical, 57 bits virtual
Byte Order:                              Little Endian
CPU(s):                                  192
On-line CPU(s) list:                     0-191
Vendor ID:                               GenuineIntel
[pip3] mypy==1.16.0
[pip3] mypy_extensions==1.1.0
[pip3] numpy==1.24.4
[pip3] nvidia-cuda-cupti==13.3.35
[pip3] oneccl==2022.0.0
[pip3] oneccl-devel==2022.0.0
[pip3] onemkl-license==2026.0.0
[pip3] onemkl-sycl-blas==2026.0.0
[pip3] onemkl-sycl-dft==2026.0.0
[pip3] onemkl-sycl-lapack==2026.0.0
[pip3] onemkl-sycl-rng==2026.0.0
[pip3] onemkl-sycl-sparse==2026.0.0
[pip3] onnx==1.21.0
[pip3] onnx-ir==0.1.16
[pip3] onnxscript==0.6.2
[pip3] optree==0.13.0
[pip3] tbb==2023.0.0
[pip3] tcmlib==1.5.0
[pip3] torch==2.14.0a0+git9d7d44e
[pip3] torchao==0.18.0.dev20260628+xpu
[pip3] torchaudio==2.11.0a0+c0cbdb9
[pip3] torchvision==0.29.0a0+f098790
[pip3] triton-xpu==3.7.2+git5fcc14d9
[pip3] umf==1.1.0
[conda] No relevant packages
</details>

--- a/test/xpu/functorch/test_control_flow_xpu.py
+++ b/test/xpu/functorch/test_control_flow_xpu.py
@@ -9002,6 +9002,7 @@ class GraphModule(torch.nn.Module):
         while_loop_stack_output = torch.ops.higher_order.while_loop_stack_output(while_loop_cond_graph_0, while_loop_body_graph_0, (primals_1,), (primals_3, primals_2));  while_loop_cond_graph_0 = while_loop_body_graph_0 = None
         getitem: "f32[u2, 3, 3]" = while_loop_stack_output[0];  while_loop_stack_output = None
         select: "f32[3, 3]" = torch.ops.aten.select.int(getitem, 0, -1)
+
         unsqueeze: "f32[1, 3, 3]" = torch.ops.aten.unsqueeze.default(primals_1, 0);  primals_1 = None
         slice_1: "f32[u2 - 1, 3, 3]" = torch.ops.aten.slice.Tensor(getitem, 0, 0, -1);  getitem = None
         cat: "f32[u2, 3, 3]" = torch.ops.aten.cat.default([unsqueeze, slice_1]);  unsqueeze = slice_1 = None

--- a/test/xpu/higher_order_ops/test_invoke_subgraph_xpu.py
+++ b/test/xpu/higher_order_ops/test_invoke_subgraph_xpu.py
@@ -2030,6 +2030,7 @@ class GraphModule(torch.nn.Module):
 
         getitem: "f32[8, 8]" = invoke_subgraph_2[0];  invoke_subgraph_2 = None
         sin: "f32[8, 8]" = torch.ops.aten.sin.default(getitem)
+
         cos: "f32[8, 8]" = torch.ops.aten.cos.default(getitem);  getitem = None
         return (sin, getitem_6, getitem_5, getitem_4, cos)
 

